app/assets/javascripts/host_edit.js

```diff
@@ -1,3 +1,5 @@
+//= require host_edit_interfaces
```

Comment:
Is this related to this patch or just a leftover from something else? If it fixes a separate issue please put it in a different PR.